src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -607,4 +654,33 @@ export class MaliciousExtensionChecker implements IWorkbenchContribution {
 	dispose(): void {
 		this.disposables = dispose(this.disposables);
 	}
-}
\ No newline at end of file
+}
+
+let SEARCH_INPUT_OPTIONS: IEditorOptions =
+{
+	fontSize: 13,
```

Comment:
There seems to be a line height issue

![image](https://user-images.githubusercontent.com/16890566/42856106-811c4416-89f8-11e8-840f-d76621a4cf6e.png)